docs/resources/waf_rule_cc_protection.md

```diff
@@ -0,0 +1,188 @@
+  "release frequency", it will be blocked.
+
+* `rate_limit_mode` - (Required, String) Specifies the rate limit mode. Valid values are as follows:
+  + **ip**: A web visitor is identified by the IP address.
+  + **cookie**: A web visitor is identified by the cookie key value.
+  + **header**: A web visitor is identified by the header key value.
+  + **other**: A web visitor is identified by the Referer field (user-defined request source).
+  + **policy**: A web visitor is identified by rule.
+  + **domain**: A web visitor is identified by domain name.
+  + **url**: A web visitor is identified by url.
+
+* `limit_num` - (Required, Int) Specifies the number of requests allowed from a web visitor in a rate limiting period.
+  The value ranges from 1 to 2,147,483,647.
+
+* `limit_period` - (Required, Int) Specifies the rate limiting period. The value ranges from 1 to 3,600 in seconds.
+
+* `block_page_type` - (Optional, String) Specifies the type of the returned page. The options are **application/json**,
+  **text/html** and **text/xml**. This parameter is valid when protective_action is set to `block` or `dynamic_block`.
+  If not specified the system default block page will be used.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project ID of WAF cc protection rule.
+
+  Changing this parameter will create a new resource.
+
+* `page_content` - (Optional, String) Specifies the content of the returned page.
+  This parameter is required when `block_page_type` has value.
+
+* `user_identifier` - (Optional, String) Specifies the user identifier.
+  This parameter is required when `rate_limit_mode` is set to `cookie` or `header`.
+  + If `rate_limit_mode` is set to `cookie`, this parameter indicates cookie name.
+  + If `rate_limit_mode` is set to `header`, this parameter indicates header name.
+
+* `other_user_identifier` - (Optional, String) Specifies the other user identifier.
+  This parameter is required when `rate_limit_mode` is set to `other`, indicates the user-defined request field.
+
+* `unlock_num` - (Optional, Int) Specifies the allowable frequency. The value ranges from 0 to 2,147,483,647.
+  This parameter is valid when `protective_action` is set to `dynamic_block`.
+
+* `lock_time` - (Optional, Int) Specifies the lock time for resuming normal page access after blocking can be set.
+  The value ranges from 0 to 65,535 in seconds. This parameter is valid when `protective_action` is set to `block`.
+
+* `request_aggregation` - (Optional, Bool) Specifies whether to enable domain aggregation statistics.
+  This parameter is valid when `rate_limit_mode` is not set to `policy`. Default to false.
+
+* `all_waf_instances` - (Optional, Bool) Specifies whether to enable global counting. Default to false.
+
+* `description` - (Optional, String) Specifies the description of WAF cc protection rule.
+
+* `status` - (Optional, Int) Specifies the status of WAF cc protection rule.
```

Comment:
```suggestion
* `status` - (Optional, String) Specifies the status of WAF cc protection rule.
```
if the type is int, cant modify to 0

Reply:
type can change to 0 with current implementation.